tljh-plasma/tljh_plasma/__init__.py

```diff
@@ -83,8 +83,7 @@ def tljh_custom_jupyterhub_config(c, tljh_config_file=CONFIG_FILE):
     c.JupyterHub.cleanup_servers = False
     c.JupyterHub.authenticator_class = PAMAuthenticator
     c.JupyterHub.spawner_class = PlasmaSpawner
-    c.JupyterHub.allow_named_servers = True
-    c.JupyterHub.named_server_limit_per_user = 2
+    c.JupyterHub.allow_named_servers = False
```

Comment:
We should be able to remove that line completely, as `allow_named_servers` default to `False`.

Reply:
You're perfectly right! Here's an excerpt from JupyterHub documentation:
>   allow_named_servers c.JupyterHub.allow_named_servers = Bool(False)

Ref: https://jupyterhub.readthedocs.io/en/stable/api/app.html

Reply:
I'll create a commit to fix this.